tests/testutil.hpp

```diff
@@ -32,7 +32,7 @@
 
 #include "../include/zmq.h"
 #include "../src/stdint.hpp"
-#include "platform.hpp"
+#include "../src/platform.hpp"
```

Comment:
Changing the platform include like this always breaks the cmake build where the plaform.hpp is generated out of source.

Can we change it back?